# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mine/maxlen.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mine-0.23/selector.c.vanilla	2023-03-31 00:20:33.878291962 +0200
+++ mine-0.23/selector.c	2023-03-31 00:25:19.834306006 +0200
@@ -34,6 +34,11 @@
 " [>]  package marked for replacing currently installed version\n"	\
 " < > / <->  dependency (not installed / marked for removal)\n"
 
+// dialog moved this macro in dlg_internals.h which isn't exposed to the end user
+#ifndef MAX_LEN
+#define MAX_LEN 2048
+#endif
+
 char ** items;
 struct directory_entry ** entries;
 int item_counter;
